### ISSUE TITLE
Cannot find name 'object' beta7

### ISSUE THREAD

**Issue by kolkov** [ui-router/angular#129]
Hi! Trying to update to beta7:
![image](https://user-images.githubusercontent.com/3740898/27319590-eaa78e10-559a-11e7-888a-85544ac55680.png)

**Comment by mkoczorowski** [comment]
I have upgraded yesterday, all working fine. Have you tried to delete node_modules and do npm cache clean and install again?

**Comment by kolkov** [comment]
Yes, I solved it already by upgrading all dependencies and reinstalling.
Thanks!

**Comment by mkoczorowski** [comment]
ok great, can we close the issue ?

**Comment by RilRil** [comment]
hello, I get the same error with the 1.0.0-rc.0.

**Comment by RilRil** [comment]
I solved it by upgrading Angular (5.0) and Typescript (2.6) \o/

**Comment by christopherthielen** [comment]
For posterity: the `object` type was added in Typescript 2.2.  Update typescript to 2.2 or higher if you're getting this error.  I've added [tests for typescript 2.2 through 2.6](https://github.com/ui-router/angular/commit/5a09592985379e83cf4942b53396ca1243b7cc8d) so in the future I'll know when changes break earlier versions of typescript.

Related issue: https://github.com/ui-router/core/issues/90#event-1385308071
